--- src/network/network.cc.orig	2021-01-20 20:56:38.000000000 +0100
+++ src/network/network.cc	2021-01-20 20:56:50.000000000 +0100
@@ -335,7 +335,7 @@
       }
     }
 
-    if ( bind( sock(), &local_addr.sa, local_addr_len ) == 0 ) {
+    if ( ::bind( sock(), &local_addr.sa, local_addr_len ) == 0 ) {
       set_MTU( local_addr.sa.sa_family );
       return true;
     } else if ( i == search_high ) { /* last port to search */
